lib/mvi/messages/find_candidate_message.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+require 'ox'
+require_relative 'message_builder'
+
+module MVI
+  module Messages
+    # Builds an MVI SOAP XML message.
+    #
+    # = Usage
+    # Call the .build passing in the candidate's first and last name, dob, and ssn.
+    #
+    # Example:
+    #  dob = Time.new(1980, 1, 1).utc
+    #  message = MVI::Messages::FindCandidateMessage.build('John', 'Smith', dob, '555-44-3333')
+    #
+    class FindCandidateMessage
+      extend MVI::Messages::MessageBuilder
+      EXTENSION = 'PRPA_IN201305UV02'
+
+      class << self
+        def build(first_name, last_name, dob, ssn)
```

Comment:
I'm not really too familiar with how this is all architected, but is it at all preferable to create Message instances that can be passed off to a Message dispatcher that's responsible for sending to the SOAP service? It'd make testing message attribute expectations a bit easier than matching against the generated XML directly.